packages/output-view/src/browser/output-view-contribution.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { MenuContribution, CommandContribution, MenuModelRegistry, CommandRegistry, Command } from "@theia/core";
+import { WidgetManager, FrontendApplication, CommonMenus } from "@theia/core/lib/browser";
+import { inject, injectable, multiInject } from "inversify";
+import { OUTPUT_VIEW_KIND, OutputViewWidget } from "./output-view-widget";
+import { LanguageClientContribution } from "@theia/languages/lib/browser";
+
+export namespace OutputViewCommands {
+    export const OPEN: Command = {
+        id: 'outputView:open',
+        label: 'Open Output View'
```

Comment:
I asked miro in the other PR to changes these kind of commands to `Toggle Output View`, so that it closes the Output view when it has currently focus.